amethyst_core/src/transform/components/transform.rs

```diff
@@ -0,0 +1,788 @@
+//! Local transform component.
```

Comment:
This file is exactly the same as from master, but has an added `parent_matrix` field.